src/jobservice/job/impl/gc/garbage_collection.go

```diff
@@ -15,7 +15,9 @@
 package gc
 
 import (
-	"fmt"
+	"github.com/goharbor/harbor/src/lib/errors"
```

Comment:
I thought we discussed keeping the old gc job as backup plan?

Reply:
I'll create another PR to handle this, maybe can use a parameter to trigger differernt job.